pkg/alertmanager/statefulset.go

```diff
@@ -366,8 +366,13 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config) (*appsv1.S
 			MountPath: secretsDir + s,
 		})
 	}
-
-	terminationGracePeriod := int64(0)
```

Comment:
you'd need to rebase your fork because the default value is currently [120s](https://github.com/prometheus-operator/prometheus-operator/blob/000315b9aaa20e228f38e74852741d383bedafe4/pkg/alertmanager/statefulset.go#L495).